src/components/Input/components/Switch/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import InputBase from '../InputBase';
+import { Props } from './types';
+
+
+function Switch({ className, ...otherProps }: Props): ReactElement {
```

Comment:
For consistency's sake, the `props` should always be multiline if there is more than one and the block should be padded with two linebreaks.